### PR TITLE
Fix paste issue with iter_stidx() documentation

### PR DESCRIPTION
**Pull request by ratmice**

### CODE REVIEW

File: lrtable/src/lib/stategraph.rs [hunk 1]
```diff
@@ -36,8 +36,8 @@ where
         self.start_state
     }
 
-    /// Return an iterator which produces (in order from `0..self.rules_len()`) all this
-    /// grammar's valid `RIdx`s.
+    /// Return an iterator which produces (in order from `StorageT::zero()..self.all_states_len()`)
+    /// all this grammar's valid `StIdx`s.
```
Comment (ltratt) [inline lrtable/src/lib/stategraph.rs:40]: The code below says `self.states.len()`?

Comment (ratmice) [reply lrtable/src/lib/stategraph.rs:40]: Sorry, I hadn't noticed the other uses of `self.states.len()`, my thinking was that the `states` field is private, while `all_states_len()` is public, however it brings with it the `zero()` weirdness.

Could change it to match the others, or vice versa. Whichever you prefer

Comment (ltratt) [reply lrtable/src/lib/stategraph.rs:40]: Good spot -- we should only document using the external API!

**Pull request merged**